reference/release-notes/1.23.1.md

```diff
@@ -0,0 +1,129 @@
+  - The [HTTP origin](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Origin) (`core.https_allowed_origin`)
+  - List of allowed HTTP headers (`core.https_allowed_headers`)
+  - List of allowed HTTP methods (`core.https_allowed_methods`)
+* You can skip setting up coturn when initialising the appliance and can configure a custom STUN server.<!--AC-2719-->
+* The [`prepare-node-script`](prepare-node-script.md) command is extended to support systems with an already installed NVIDIA driver. <!--AC-2714-->
+* You can now set up a custom identity provider for the dashboard by configuring an OpenID Connect provider through the preseed configuration when initialising the appliance.
+
+### Streaming
+
+* With the 1.23.0 release, the functionalities of the *Instances* page and the *Sessions* page on the web dashboard were merged. For debugging purposes, viewing a list of available sessions can be really useful. With the 1.23.1 release, you can list all available sessions using the `anbox-stream-gateway sessions list` command.<!--AC-2662-->
+* You can dynamically change the display density when joining an existing streaming session.<!--AC-2673-->
+
+### VHAL
+
+* Starting with 1.23.0, the VNDK version required for building a custom VHAL is 34.<!--AC-2720-->
+
+### Android vendor image
+
+* WiFi support for Android 12 vendor images.<!--AC-2707-->
+
+### Other
+
+* Android security updates for September 2024 (see [Android Security Bulletin - September 2024](https://source.android.com/docs/security/bulletin/2024-09-01) for more information). <!--AC-2712-->
+* The Android WebView has been updated to [128.0.6613.127](https://chromereleases.googleblog.com/2024/09/chrome-for-android-update.html).
+
+## Known issues
+
+* Since version 1.23.0, the Anbox WebRTC Data Proxy service starts on demand rather than at container startup. However, its startup time may take longer than expected, which can negatively impact the {ref}`out-of-band data exchange <sec-oob-data-version-2>` between the WebRTC server and client. This issue can be worked around by applying the following tweak in a [pre-start hook](https://anbox-cloud.io/docs/ref/hooks).
+
+  ```
+  #!/bin/sh -ex
+  if  [ "$CONTAINER_TYPE" = "regular" ]; then
+    exit 0
+  fi
+
+  DROP_IN_DIR=/etc/systemd/system/anbox-webrtc-data-proxy.service.d/
+  sudo mkdir -p "${DROP_IN_DIR}"
+  sudo tee "${DROP_IN_DIR}/override.conf" > /dev/null <<EOF
+  [Install]
+  WantedBy=multi-user.target
+  EOF
+
+  sudo systemctl daemon-reload
+  sudo systemctl enable anbox-webrtc-data-proxy
+  ```
+* For the Anbox Cloud Appliance deployment, after it is initialised, you may encounter the following error when creating an application or launching an instance:
```

Comment:
Should we remove this? We can wait for @adglkh and confirm tomorrow.

Reply:
Thanks for the head up. @morphis gave it a try with 24.04 ubuntu image on the AWS last Friday an couldn't reproduce this one. And I gave it another try this morning with 22.04 ubuntu image and failed to reproduce this issue either now. Since this issue is not reproducing, I'd suggest removing it for now. And I'll keep an eye on this one.